buildscripts/checkstyle.xml

```diff
@@ -144,7 +144,9 @@
             <property name="forbiddenSummaryFragments"
              value="^@return the *|^This method returns |^A [{]@code [a-zA-Z0-9]+[}]( is a )"/>
         </module>
-        <module name="JavadocParagraph"/>
+        <module name="JavadocParagraph">
+            <property name="allowNewlineParagraph" value="false"/>
```

Comment:
What's the motivation here?

Reply:
Example of javadoc that fails the build without this. See the line with the lone `<p>` before the `<ul>` - weaver seems to use `<p> ` as a line break. 
```
public final class ErrorAttributes {
  /**
   * Describes a class of error the operation ended with.
   *
   * <p>Notes:
   *
   * <p>The {@code error.type} SHOULD be predictable, and SHOULD have low cardinality.
   *
   * <p>When {@code error.type} is set to a type (e.g., an exception type), its canonical class name
   * identifying the type within the artifact SHOULD be used.
   *
   * <p>Instrumentations SHOULD document the list of errors they report.
   *
   * <p>The cardinality of {@code error.type} within one instrumentation library SHOULD be low.
   * Telemetry consumers that aggregate data from multiple instrumentation libraries and
   * applications should be prepared for {@code error.type} to have high cardinality at query time
   * when no additional filters are applied.
   *
   * <p>If the operation has completed successfully, instrumentations SHOULD NOT set {@code
   * error.type}.
   *
   * <p>If a specific domain defines its own set of error identifiers (such as HTTP or gRPC status
   * codes), it's RECOMMENDED to:
   *
   * <p>
   *
   * <ul>
   *   <li>Use a domain-specific attribute
   *   <li>Set {@code error.type} to capture all errors, regardless of whether they are defined
   *       within the domain-specific set or not.
   * </ul>
   */
```

Reply:
Yes - that's the motivation. I asked on the one CNCF slack channel about this, but didn't hear any feedback.

I'm unable to figure out how to resolve this succesfully, weaver is using <p> and the actual underlying data in the YAML has this empty line break.

Reply:
> I'm unable to figure out how to resolve this succesfully, weaver is using

Looks like some of the message got truncated

FWIW, I don't think its a big deal to have empty paragraphs.

Reply:
Great - I'd like to leave the changes as-is then.